### DISCUSSION TITLE
Ethcon Korea 2023 Sponsorship

### DISCUSSION THREAD

**Discussion by diamondnegroni** [ubiquity/business-development#59] [General]
<html>

[Ethcon_Korea_2023_Sponsorship_Abstract (1).pdf](https://github.com/ubiquity/business-development/files/12044187/Ethcon_Korea_2023_Sponsorship_Abstract.1.pdf)

<body>
<p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><strong style="box-sizing: border-box; font-weight: var(--base-text-weight-semibold, 600);">Ethcon Korea 2023 Sponsorship</strong></p><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">I am proposing a strategic combination of options for Ethcon Korea 2023 sponsorship to maximize our global exposure and make the most of this opportunity.</p><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Ethcon Korea which is held between Sep. 1-3 is a significant event as it marks the first Ethcon event in Korea since the COVID-19 pandemic, and it is closely followed by KBW 2023, taking place from Sep. 4-10. Given these circumstances, we anticipate a large turnout of global participants at Ethcon Korea. Moreover, the sponsorship cost for this event is relatively competitive compared to other similar events. For instance, the minimum cost of ETH Denver 2023 Sponsorship was $15,000.</p><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Considering that Ethcon Korea will be Ubiquity's inaugural offline presence, it is crucial for us to utilize our resources effectively and increase our global exposure within the developer community. In light of these objectives, I propose a combination of the following options: a booth (Option 1), Branded T-shirts (Option 3), and 3 models (Option 4).</p><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">This strategic combination will not only provide us with a physical presence at the event through the booth but also enhance our visibility with the Branded T-shirts, which will be worn by event attendees. Additionally, having 3 models representing our brand will create a strong visual impact and further increase our exposure among the attendees. By choosing this combination of options, we will be able to maximize our resources and effectively promote our presence at Ethcon Korea, resulting in increased global exposure for the development pool.</p><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Ethcon Korea 2023<span> </span><a href="https://ethcon.kr/" rel="nofollow" style="box-sizing: border-box; background-color: transparent; color: var(--fgColor-accent, var(--color-accent-fg)); text-decoration: none;">https://ethcon.kr/</a></p><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Ethereum Developer Conference &amp; Hackathon<span> </span><a href="https://medium.com/ethconkorea/announcing-ethcon-korea-2023-75c8605bd18c" rel="nofollow" style="box-sizing: border-box; background-color: transparent; color: var(--fgColor-accent, var(--color-accent-fg)); text-decoration: none;">https://medium.com/ethconkorea/announcing-ethcon-korea-2023-75c8605bd18c</a></p><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Date: September 1~3(Fri-Sun), 2023</p><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Venue: Platz2, Seoul,<span> </span><a href="https://map.naver.com/v5/entry/place/1122346727?placePath=%2Fphoto" rel="nofollow" style="box-sizing: border-box; background-color: transparent; color: var(--fgColor-accent, var(--color-accent-fg)); text-decoration: none;">https://map.naver.com/v5/entry/place/1122346727?placePath=%2Fphoto</a></p><p dir="auto" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(31, 35, 40); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Conference and Hackathon</p>

  | Hackathon | Conference
-- | -- | --
Estimated Participants | 100~200 builder | 500+ attendance
Day 1 | Registration Hackathon Begins at 5PM | Registration, Keynote Speech
Day 2 | Hackathon goes on, Education Session | No Conference Day
Day 3 | Hackathon Ends Quadratic Voting Stars | Demo Day

</body>
</html>

**Comment by diamondnegroni** [comment]
Hackathon Track Award & Conference Sponsorship
- [Conference Sponsorships](https://github.com/ubiquity/ubiquity-dollar/assets/136157002/a8aa4dce-4cca-46d6-8b60-0e450b829708)
- [Hackathon Sponsorship table](https://github.com/ubiquity/ubiquity-dollar/assets/136157002/b700a593-a888-4b5a-85ef-357a950b55cb)

**Comment by diamondnegroni** [comment]
Four ways to promote Ubiquity Dev Pool through Sponsorship and non-Sponsorship

  
|                        |     Option 1                                                                                                                                                                            |     Option 2                                                                                       |     Option 3                                                                                              |     Option 4     (No Sponsorship)                                                                                                                                                                                                                         |
|------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
|     Item               |     Booth at Conference     Booth size 1.8m x 1.8mx                                                                                                                                     |     Educational Speaker Session at Hackathon                                                       |     200 Branded T-Shirt                                                                                   |     3 Models with     Dev Pool QR code T-shirt                                                                                                                                                                                                            |
|     Price              |     3Eth     (includes booth design)                                                                                                                                                    |     2Eth                                                                                           |     3 Eth    (Design cost included)                                                                       |     Model $300-$400 per day     1 model x 2.5 days= max $1,000     3 models = $3,000     T-shirt max $50 / shirt     Design $200-$300     Max Total budget $3,500                                                                                         |
|     Available slots    |     15                                                                                                                                                                                  |     7                                                                                              |     1                                                                                                     |     n/a                                                                                                                                                                                                                                                   |
|     Duration           |     2.5 days                                                                                                                                                                            |     30 mins                                                                                        |     n/a                                                                                                   |     2.5 days                                                                                                                                                                                                                                              |
|     Note               |     Booth located in between two buildings   where hackathon and conference is held    People need to pass by to go to rest rooms.     The project can demonstrate the Devpool          |     Company introduction     Devpool Demostration                                                  |     200 T-Shirt with your Branded T-Shirt for   Hackathon Participants                                    |     3 models walking around conference and   hackathon with Dev Pool QR Code.      Combining with referral program that prospectives   use model’s name as a referral to onboard and finish a bounty will get   additional reward and so as the model.    |
|     Pros               |     Promote, onboard, and interact at the   same time                                                                                                                                   |     Global PR opportunity      Promote   Ubiquity DAO and the Devpool     Demonstrate DevPool      |     200 multi-national participants will   wear QR Code shirts which can increase the global exposure     |     Given 95% of participants at Hackathon   is male, female model will attract participants.                                                                                                                                                             |
|     Cons               |     Need people to stay at booth.      Targeting conference participants and   Hackathon participants                                                                                   |     Risk of doxing in public     Not sure how effective it is to give   speech during hackathon    |     A bit expensive     $30 / shirt                                                                       |     The output of each model can vary. We need   to train or give a basic tutorial on dev pool to models.

**Reply by 0x4007** [reply]
## Option 2 

Seems interesting. We would need to prepare the seminar materials. Dox might be ok in this context, just have to think through this a little more.

## Option 4

![image](https://github.com/ubiquity/business-development/assets/4975670/44dcb76e-8eaf-40f4-92b2-327aac6951c4)

"UbiquiChicks"

> The output of each model can vary. We need to train or give a basic tutorial on dev pool to models.

We don't necessarily need to hire "models" we can just hire good looking and smart girls. We can just ask our friends for this I think. Models usually aren't the sharpest tools in the shed and might be significantly more expensive. I think it really comes down to the models in question.

I think it would be really funny to imply that the QR code goes to their OnlyFans or instagram or something and then it just goes to the [DevPool Flow](https://dao.ubq.fi/devpool-flow) page lol.

> 3 models walking around conference and hackathon with Dev Pool QR Code. Combining with referral program that prospectives use model’s name as a referral to onboard and finish a bounty will get additional reward and so as the model.

Ideally if we can make the compensation model a competition we can lower base pay significantly and just ask all of our lady friends if they can help us out. More volume means we can cover more leads. As for tracking who brought in who, we make the devs "Sign in with GitHub" on a custom web app we funnel them to, to capture their usernames and associate it with the QR code. But signing in with GitHub and giving that data might make some folks uncomfortable potentially, which might stop conversions. In order to push through this, we should think of a reward/incentive (perhaps a lottery?)

@Draeieg any ideas on this architecture 

## Option 5 

We should also figure out if there's any way to convince them to use our infra to facilitate with the actual execution of the hackathon. That could be a very creative Option 5. We might even be able to white label the bot to be like "ethcon bot"

@Draeieg rfc

**Reply by diamondnegroni** [reply]
Option 2 - What kinds of topics would you like to cover during 30 mins ?
Option 4 - Asking friends is also an option but if you are gonna put "onlyfans link" i'm not sure how they are going to respond to that. If we are going to use that many people. I will talk to Jenny to see how many Ubiquity people they can let us in to the hackathon and the conference. As per tracking the referrals, can we make it as simple as possible ? for example, instead of having users to input referral on the onboarding page, can we ask the user to put referrals when they are claming or assging tasks?

**Reply by 0x4007** [reply]
> Option 2 - What kinds of topics would you like to cover during 30 mins ?

I just realized that I had a typo, but we would need to prepare these materials. I'm not sure yet.

> instead of having users to input referral on the onboarding page, can we ask the user to put referrals when they are claming or assging tasks?

Unless there's a direct incentive, the system will be dysfunctional. I guarantee that more than half will not follow that request, unless if they are incentivized to do so.

**Comment by diamondnegroni** [comment]
ETH Denver Sponsorship Table for a reference point
->The cost of minimum sponsorship at Ethcon Korea is 3 ETH which shows how the sponsorship at Ethcon Korea is cost-effective


- [ETH Denver Sponsorship Table](https://github.com/ubiquity/business-development/assets/136157002/0f8bc0ee-59aa-424c-b604-d7ee7af28e20)

**Comment by 0x4007** [comment]
I think some backend tasks to handle for this are that we should 
1. boost bounty rates (perhaps we can funnel everybody to a single repo) and 
2. make a more friendly landing page to onboard new devs.

**Reply by diamondnegroni** [reply]
Yes I agree, It will be nice to have a visually appealing landing page that can correlate with Ethoncon Korea
What if we boost significantly bounty rates during hackathon period ?

**Comment by diamondnegroni** [comment]
July 16. meeting summary

Option 1 
How to differentiate the booth 
-Cocktails options (if not, juice)
-Good Looking girls, Ballons, Projectors 
-6 stools (can we situate 6 seats-check  ? Check with Jenny on the configuration)
-equipment: 32-inch monitor, screen capture Video of GitHub, something to get the attention,
-Training: Lena, Jenny, to get an intro to dev people

Option 2 	
-Topics to discuss: background, Decentralization, ETH Ethos, DAOs, 
Doxing: Lead Developer relation at Ubiquity Devpool
-List of things to ask Jenny
-What kinds of speakers are signed up?
-How are the seating in the speaker?  and in what Configuration?
-Is the educational session will be live on Youtube?

Option 3 
-200 Customized bags -15 inch laptop size
- Developer fashion trend – utility focus, nothing fancy, i.e. ) bracelet 

Option 4
-A hunter for C-level people at projects for the Partnership 
-Ideal candidate: English speaking, understanding of blockchain, Github friendly, developer, good-looking, and outgoing personality

**Reply by 0x4007** [reply]
A lottery for completing a bounty that ethcon participants are uniquely eligible to receive. 200 extra DAI.

- provides an incentive for them to go through our landing page to sign in with GitHub

**Reply by diamondnegroni** [reply]
Q&A with Jenny  (July. 17)

1. Alcohol Policy _ Will be determined on July 23 from the Ethcon Organizer Meeting
2. 30 mins Speaker session - The list of confirmed speakers are Vitaliv, Optimism, Taiko(ZKrollup), Chainlink
                               -There will be rugs on the floor where people can sit by the stage
                               -The speaker session will be broadcasted on-live overseas
3. Hackathon Theme - will be shared once it's finalized 
4. Contact info of participants to Sponsors - will be complicated as the Ethcon needs permission from the participants 
5. 15 inch laptop back pack - will review it get back to us this week

**Comment by 0x4007** [comment]
@diamondnegroni perhaps you can write out the latest updated plan spanning across all of the initiatives e.g. bar, bag, booth, and babes
